irictl-machine/cmd/irictl-machine/irictlmachine/get/event/event.go

```diff
@@ -17,11 +18,15 @@ import (
 )
 
 type Options struct {
-	Labels map[string]string
+	Labels         map[string]string
+	EventsFromTime string
+	EventsToTime   string
 }
 
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringToStringVarP(&o.Labels, "labels", "l", o.Labels, "Labels to filter the events by.")
+	fs.StringVarP(&o.EventsFromTime, "eventsFromTime", "f", o.EventsFromTime, fmt.Sprintf("Events From Time to filter the events by. In the format of %s", time.DateTime))
```

Comment:
Can we use a `Duration` here and default it to some value?

Reply:
Done added duration with default value of 60 minutes